src/cascade/executor/epiviz_runner.py

```diff
@@ -79,6 +80,8 @@ def add_mortality_data(model_context, execution_context):
     )
     csmr["measure"] = "mtspecific"
     csmr = csmr.rename(columns={"location_id": "node_id"})
+    # Ensure that the csmr index doesn't overlap with the existing seq values in the observations.
+    csmr.index = csmr.index.values + model_context.input_data.observations.index.max() + 1
```

Comment:
This is the main comment... and you're right. This was uncomfortable, which is a sign. I think I need the seq as an index for joins. It should be a column, though, so it can't be lost. And we need to keep `hold_out` as a column too instead of using `constraints`. Let me make a commit and then see what I can do.